src/lib/mongoBackend/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ SET (SOURCES
     mongoGetSubscriptions.cpp
     connectionOperations.cpp
     mongoSubCache.cpp
-    safeBsonGet.cpp
+    safeMongo.cpp
```

Comment:
The module now includes new functions to make driver usage safer not related with BSON, thus I have changed to a more general name.

NTC